rust/stackable-cockpit/src/xfer/cache.rs

```diff
@@ -97,15 +109,14 @@ impl Cache {
                 let mut entries = fs::read_dir(base_path).await.context(CacheIoSnafu)?;
 
                 while let Some(entry) = entries.next_entry().await.context(CacheIoSnafu)? {
-                    files.push((
-                        entry.path(),
-                        entry
-                            .metadata()
-                            .await
-                            .context(CacheIoSnafu)?
-                            .modified()
-                            .context(CacheIoSnafu)?,
-                    ))
+                    let metadata = entry.metadata().await.context(CacheIoSnafu)?;
+
+                    // Skip the last-auto-purge file
+                    if metadata.is_file() && is_protected_file(entry.file_name())? {
```

Comment:
Is there a reason to limit this to regular files?

Reply:
Not really. We can remove this check to allow protected dirs in the future (if ever needed).

Reply:
Removed in b51571d